src/core.js

```diff
@@ -442,15 +450,9 @@ jQuery.extend( {
 	support: support
 } );
 
-// JSHint would error on this code due to the Symbol not being defined in ES5.
```

Comment:
It was put here because I wanted to make sure all `Symbol` uses are guarded by a typeof check to not break ES5-only browsers. Could we just change the `jshint ignore` here to `/* eslint-disable no-undef */` & `/* eslint-enable no-undef */`? I assume there's no way to declare a global to exist only between 2 pragmas?